### DISCUSSION TITLE
what is `celery/examples/eventlet/bulk_task_producer.py` for?

### DISCUSSION THREAD

**Discussion by NoraGithub** [celery/celery#7576] [Q&A] [answered]
i do not see anywhere use  this file? is it really an example of how eventlet use?

[a comment by an automated account is not shown]

**Comment by Sauraj1234** [comment, marked as answer]
@NoraGithub I dont think this file is used in Celery anywhere 
its just the example , Anyone who has more experienced  Correct me if i am wrong 
Please readme.rst  inside (celery/examples/eventlet)

**Comment by auvipy** [comment]
don't raise issues without being confirmed, so first start a discussion.

**Reply by NoraGithub** [reply]
my bad, and i apologize for that

**Reply by auvipy** [reply]
apology would be accepted if you try to contribute something meaningful to celery!

**Comment by Mahir-Panchal** [comment]
Opened PR #10399 to add a docstring and README entry for this file, since it looks like it was never actually documented after this was raised.
